Tests/scripts/lock_cloud_machines.py

```diff
@@ -7,12 +7,14 @@
 import requests
 from google.cloud import storage  # noqa
 import argparse
+from Utils.github_workflow_scripts.utils import get_env_var
 
+GITLAB_SERVER_URL = get_env_var('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
 LOCKS_BUCKET = 'xsoar-ci-artifacts'
 QUEUE_REPO = 'queue'
 MACHINES_LOCKS_REPO = 'machines_locks'
-JOB_STATUS_URL = 'https://code.pan.run/api/v4/projects/{}/jobs/{}'  # disable-secrets-detection
-CONTENT_GITLAB_PROJECT_ID = '2596'
+JOB_STATUS_URL = '{}/api/v4/projects/{}/jobs/{}'  # disable-secrets-detection
+CONTENT_GITLAB_PROJECT_ID = get_env_var('CI_PROJECT_ID', '2596')  # the default is the id of the content repo in code.pan.run
```

Comment:
When the migration is over we'd need to change this default value.
(Add this to the Phase_2 issue.)